Css/navbar.css

```diff
@@ -1,3 +1,11 @@
-.active {
+.active
+{
   background-color: rgba(119, 119, 119, 0.24);
+}
+
+.searchBarInvalid
+{
+  border-color: darkred;
+  border: 10px, 10px, 10px, 10px;
+  color: black;

```

Comment:
No need for this CSS remove, use bootstrap "is-invalid"
URL: https://mdbootstrap.com/docs/jquery/forms/validation/